Buildings/ThermalZones/Detailed/Examples/ISAT/Tutorial/MixedConvection.mo

```diff
@@ -0,0 +1,337 @@
+within Buildings.ThermalZones.Detailed.Examples.ISAT.Tutorial;
+model MixedConvection "Tutorial for the mixed convection case"
+  extends Modelica.Icons.Example;
+  package MediumA = Buildings.Media.Air (
+        T_default=283.15) "Medium model";
+  parameter Integer nConExtWin=0 "Number of constructions with a window";
+  parameter Integer nConBou=0
+    "Number of surface that are connected to constructions that are modeled inside the room";
+  parameter Integer nSurBou=6
+    "Number of surface that are connected to the room air volume";
+  parameter Integer nConExt=0
+    "Number of exterior constructions withour a window";
+  parameter Integer nConPar=0 "Number of partition constructions";
+  Modelica.Blocks.Sources.Constant qRadGai_flow(k=0) "Radiative heat gain"
+    annotation (Placement(transformation(extent={{-40,20},{-20,40}})));
+  Modelica.Blocks.Sources.Constant qConGai_flow(k=0) "Convective heat gain"
+    annotation (Placement(transformation(extent={{-40,-20},{-20,0}})));
+  Modelica.Blocks.Sources.Constant qLatGai_flow(k=0) "Latent heat gain"
+    annotation (Placement(transformation(extent={{-40,-60},{-20,-40}})));
+  Modelica.Blocks.Routing.Multiplex3 multiplex3_1
+    annotation (Placement(transformation(extent={{20,-20},{40,0}})));
+  BoundaryConditions.WeatherData.ReaderTMY3 weaDat(filNam=
+    Modelica.Utilities.Files.loadResource("modelica://Buildings/Resources/weatherdata/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"),
+    TDryBul=293.15)
+    annotation (Placement(transformation(extent={{140,80},{160,100}})));
+  Buildings.ThermalZones.Detailed.ISAT roo(
+    redeclare package Medium = MediumA,
+    surBou(
+     name={"East Wall","West Wall","North Wall","South Wall","Ceiling","Floor"},
+     A={0.9,0.9,1,1,1,1},
+     til={Buildings.Types.Tilt.Wall,Buildings.Types.Tilt.Wall,
+        Buildings.Types.Tilt.Wall,Buildings.Types.Tilt.Wall,
+        Buildings.Types.Tilt.Ceiling,Buildings.Types.Tilt.Floor},
+     each absIR=1e-5,
+     each absSol=1e-5,
+     each boundaryCondition=Buildings.ThermalZones.Detailed.Types.CFDBoundaryConditions.Temperature),
+    lat = 0.012787839282646,
+    AFlo = 1*1,
+    hRoo = 1,
+    linearizeRadiation = false,
+    useCFD = true,
+    haveSource=false,
+    nSou=0,
+    sensorName = {"Occupied zone air temperature", "Velocity"},
+    cfdFilNam = "modelica://Buildings/Resources/Data/ThermalZones/Detailed/Examples/ISAT/Tutorial/MixedConvection/input.ffd",
+    nConExt = nConExt,
+    nConExtWin = nConExtWin,
+    nConPar = nConPar,
+    nConBou = nConBou,
+    nSurBou = nSurBou,
+    nPorts=2,
+    portName={"Inlet","Outlet"},
+    samplePeriod = 200)
+  annotation (Placement(transformation(extent={{80,-38},{120,2}})));
+  HeatTransfer.Sources.FixedTemperature TOthWal[nSurBou-1](each T=283.15)
+    "Temperature for other walls"          annotation (Placement(transformation(
```

Comment:
change the annotation to another line and re-organize the text

Reply:
Revised.